src/mnemonic/mnemonic.py

```diff
@@ -65,28 +66,26 @@ def b58encode(v: bytes) -> str:
 
 
 class Mnemonic(object):
-    def __init__(self, language: str):
+    def __init__(self, language: str = "english"):
         self.language = language
         self.radix = 2048
-        with open(
-            "%s/%s.txt" % (self._get_directory(), language), "r", encoding="utf-8"
-        ) as f:
-            self.wordlist = [w.strip() for w in f.readlines()]
-        if len(self.wordlist) != self.radix:
-            raise ConfigurationError(
-                "Wordlist should contain %d words, but it contains %d words."
-                % (self.radix, len(self.wordlist))
-            )
-
-    @staticmethod
-    def _get_directory() -> str:
-        return os.path.join(os.path.dirname(__file__), "wordlist")
+        f = os.path.join(os.path.dirname(__file__), f"wordlist/{language}.txt")
+        if os.path.exists(f) and os.path.isfile(f):
+            with open(f, "r", encoding="utf-8") as f:
+                self.wordlist = [w.strip() for w in f.readlines()]
+            if len(self.wordlist) != self.radix:
+                raise ConfigurationError(
+                    "Wordlist should contain %d words, but it contains %d words."
+                    % (self.radix, len(self.wordlist))
+                )
```

Comment:
Since you are using format strings in this PR, please convert this too.

Reply:
Done.